components/NewTab.vue

```diff
@@ -0,0 +1,56 @@
+<script setup lang="ts"></script>
+<template>
+  <div>
+    <img src="~/assets/images/readingBookIllustration.jpeg" />
+    <h1>Start your Journey with Books</h1>
+    <h2>Let's get you Started</h2>
+    <button class="subscribe-btn">Subscribe Now $99</button>
+    <p>Or</p>
+    <button class="later-btn">Maybe Later</button>
+  </div>
+</template>
+<style lang="scss" scoped>
+@import "assets/sass/main.scss";
```

Comment:
Why have you imported the `main.scss `file here? It's already declared in the `nuxt.config`.